docs/understanding-tracking-design/versioning-your-data-structures/enterprise/index.md

```diff
@@ -0,0 +1,32 @@
+---
+title: "Versioning Data Structures in BDP Enterprise"
+date: "2023-03-01"
+sidebar_position: 20
+---
+
+## How do I version?
+
+### Breaking and non-breaking changes
+
+In Data Structures UI at the point of publishing a schema you'll be asked to select which version you'd like to create. There are two options:
+
+- **Non-breaking** - a non-breaking change is backward compatible with historical data and increments the `patch` number i.e. `1-0-0` -> `1-0-1`.
```

Comment:
This explanation seems like it could work for both Cloud and Enterprise. Why duplicate it?

Reply:
We don't offer patching in Cloud. And you don't get asked what version to pick. It's a different workflow.